Until we have eventfd, we need to link with epoll-shim.

Index: Source/WebCore/CMakeLists.txt
--- Source/WebCore/CMakeLists.txt.orig
+++ Source/WebCore/CMakeLists.txt
@@ -2225,6 +2225,7 @@ set(WebCore_LIBRARIES
     LibXml2::LibXml2
     SQLite3::SQLite3
     ZLIB::ZLIB
+    "epoll-shim"
 )
 set(WebCore_FRAMEWORKS
     JavaScriptCore
